lib/core/src/stringOpr.cpp

```diff
@@ -58,7 +58,7 @@ char *rstrcpy( char *dest, const char *src, int maxLen ) {
     }
     else if ( status >= 0 ) {
         rodsLog( LOG_ERROR, "rstrcpy not enough space in dest, slen:%d, maxLen:%d", status, maxLen );
-        rodsLog( LOG_DEBUG, "rstrcpy arguments dest [%s] src [%s]", dest, src );
+        rodsLog( LOG_ERROR, "rstrcpy arguments dest [%s] src [%s]", dest, src );
```

Comment:
should we just... show src?  we really don't care about dest if it didn't work, right?

Reply:
Might be nice to have if some existing code uses `dest` without checking for errors in copying.

Reply:
fair enough.   any reason not to combine the two lines into a single error message?

Reply:
Nope! Can do though.